### PR TITLE
[BQ] modal for getting bq sql

### PR DESCRIPTION
**Pull request by chejennifer**
screen-recording-_25_
screen-recording-_26_

### CODE REVIEW

File: static/js/tools/shared/bq_modal.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      <ModalHeader toggle={() => setModalOpen(false)}>
+        Analyze this data in BigQuery
+      </ModalHeader>
+      <ModalBody>
+        <div className="big-query-sql-instructions">
+          <div>
+            To run this query, first{" "}
+            <a
+              href={
+                "https://console.cloud.google.com/bigquery/analytics-hub/exchanges(analyticshub:projects/841968438789/locations/us/dataExchanges/data_commons_17d0b72b0b2/listings/data_commons_1803e67fbc9)?project=datcom-store"
```
Comment (pradh) [inline static/js/tools/shared/bq_modal.tsx:79]: Can we drop the `?project=datcom-store` part?

**Review by beets — APPROVED** [review]
this is amazing! nice change!

**Review by pradh — APPROVED** [review]
This is awesome!

File: static/js/tools/shared/bq_modal.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/shared/bq_modal.tsx]: done

**Pull request merged**